coderd/audit/request.go

```diff
@@ -238,3 +248,30 @@ func parseIP(ipStr string) pqtype.Inet {
 		Valid: ip != nil,
 	}
 }
+
+type GroupMemberLists struct {
+	OldGroupMembers []string
+	NewGroupMembers []string
+}
+
+// Adds a 'members' key to Group resource diffs
+// in order to capture the addition or removal of group members
+func addGroupMemberDiff(logCtx context.Context, diff Map, groupMemberLists json.RawMessage, logger slog.Logger) Map {
```

Comment:
The context should normally just be called `ctx`, and the context + logger should always be the first and second fields.

```suggestion
func addGroupMemberDiff(ctx context.Context, logger slog.Logger, diff Map, groupMemberLists json.RawMessage) Map {
```